src/Microsoft.Azure.WebJobs.Host/Executors/FunctionInvoker.cs

```diff
@@ -48,16 +49,26 @@ public IReadOnlyList<string> ParameterNames
             get { return _parameterNames; }
         }
 
+        public object Instance
+        {
+            get
+            {
+                if (_instance == null)
+                {
+                    _instance = _instanceFactory.Create();
+                }
+                return _instance;
+            }
+        }
```

Comment:
Why is this being cached?  Before, each call to InvokeAsync() gives a new instance. Now, they'll share.